cmd/bnsd/app/app_performance_test.go

```diff
@@ -22,9 +21,9 @@ func BenchmarkBnsdEmptyBlock(b *testing.B) {
 
 	type dict map[string]interface{}
 	genesis := dict{
-		"gconf": map[string]interface{}{
-			cash.GconfCollectorAddress: hex.EncodeToString(aliceAddr),
-			cash.GconfMinimalFee:       coin.Coin{}, // no fee
+		"cashconf": cash.Configuration{
```

Comment:
Same here, bad genesis files.
I guess we never ran test benchmarks here....

Reply:
Yup.

`go test -bench=. ./cmd/bnsd/app` fails.
Please fix the tests there

Reply:
done in https://github.com/iov-one/weave/pull/503/commits/a8e36ccf908854b892bfa98c31c3f33dbb116505